.circleci/config.yml

```diff
@@ -10,16 +10,27 @@ jobs:
       - image: circleci/ruby:2.5.1-node
         environment:
           BUNDLER_VERSION: 2.0.1
+          PGHOST: 127.0.0.1
+          PGUSER: omer.katzir
+          RAILS_ENV: test
```

Comment:
out of curiosity - isn't this the default when running inside CircleCI image?

Reply:
tried running without these fields and failed on DB creation